README.md

```diff
@@ -2,8 +2,26 @@
 
 ![License](https://img.shields.io/github/license/hackclub/burrow) ![Apple Build Status](https://img.shields.io/github/actions/workflow/status/hackclub/burrow/build-apple.yml?branch=main&label=macos%2C%20ios&logo=Apple) ![Crate Build Status](https://img.shields.io/github/actions/workflow/status/hackclub/burrow/build-rust.yml?branch=main&label=crate&logo=Rust)
 
-Burrow is a tool for burrowing through firewalls, built by teenagers at [Hack Club](https://hackclub.com/).
+Burrow is an open source tool for burrowing through firewalls, built by teenagers at [Hack Club](https://hackclub.com/).
 
-At its core, `burrow` is a command line utility written in Rust that can open virtual interfaces and direct traffic through them.
+`burrow` provides a simple command-line tool to open virtual interfaces and direct traffic through them
 
-You can use it directly from the command line, or you can use it wrapped inside of an app. Currently, there are apps for iOS and macOS that you can use to start the tunnel.
+## Contributing
+
+Burrow is fully open source, you can fork the repo and start contributing easily. For more information and in-depth discussions, visit the `#burrow` channel on the [Hack Club Slack](https://hackclub.com/slack/), here you can ask for help and talk with other people interested in burrow! 
+
+The project structure is divided in the following folders: 
+
+[] To do, add folder directories
+
```

Comment:
```suggestion

\```
Apple/ # Xcode project for burrow on macOS and iOS
burrow/ # Higher-level API library for tun and tun-async
tun/ # Low-level interface to OS networking
    src/
        unix/ # macOS and Linux code
        windows/ # Windows networking code
tun-async/ # Async interface to tun
\```

```

Had to escape the inner  backticks to make github keep it as a suggestion, but this will fill out this section

Reply:
Okok, I'll commit it